internal/provider/resource_ko_image.go

```diff
@@ -81,10 +78,13 @@ func doBuild(ctx context.Context, ip string) (string, error) {
 	return ref.String(), nil
 }
 
-const koDockerRepo = "gcr.io/jason-chainguard"
-
 func resourceKoBuildCreate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
-	ref, err := doBuild(ctx, d.Get("importpath").(string))
+	repo, ok := meta.(string)
```

Comment:
Why is `meta` expected to be a string here? If there were other params in the provider, how would we get those here? Or is this just magic because the provider only has one param?